<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QFrame" name="frmLeft">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="boxHost">
         <property name="title">
          <string>Host</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="txtHostName">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>1000</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>HostName</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox">
            <property name="suffix">
             <string/>
            </property>
            <property name="prefix">
             <string>Port: </string>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>1234</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnConnect">
            <property name="text">
             <string>Connect</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnStartStream">
            <property name="text">
             <string>Start</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="boxStream">
         <property name="title">
          <string>View</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0">
          <item row="1" column="0" colspan="3">
           <widget class="QSlider" name="sliderZoom">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Zoom</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDial" name="dialYRot">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximum">
             <number>360</number>
            </property>
            <property name="notchTarget">
             <double>36.000000000000000</double>
            </property>
            <property name="notchesVisible">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QDial" name="dialZRot">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximum">
             <number>360</number>
            </property>
            <property name="notchTarget">
             <double>36.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QComboBox" name="cboViewType">
            <item>
             <property name="text">
              <string>Depth</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Disparity</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IR</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>RGB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Overlay</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Mode:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QDial" name="dialXRot">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximum">
             <number>360</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
            <property name="notchTarget">
             <double>36.000000000000000</double>
            </property>
            <property name="notchesVisible">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Debug</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>FPS:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Timeouts:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="Timeouts">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="FPS">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="CloudOpenGLPanel" name="canvas" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1020</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CloudOpenGLPanel</class>
   <extends>QWidget</extends>
   <header>CloudOpenGLPanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnConnect</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onConnectPressed(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>703</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStartStream</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onStartPressed(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>760</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cboViewType</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>onModeEdited(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>639</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onConnectPressed(bool)</slot>
  <slot>onStreamTypeChanged(int)</slot>
  <slot>onResolutionChanged(int)</slot>
  <slot>onStartPressed(bool)</slot>
  <slot>onScalingChanged()</slot>
  <slot>onFPSChanged(int)</slot>
  <slot>onModeEdited(QString)</slot>
 </slots>
</ui>
